### PR TITLE
[1.16] support manifest lists

### PR DESCRIPTION
**Pull request by nalind**
Vendor in WIP branches for manifest list support to let CI exercise the changes. Basically, this is #2775 and #2852 for 1.16.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by mrunalp** [comment]
needs rebase

**Comment by mrunalp** [comment]
@nalind could you rebase this?

**Comment by saschagrunert** [comment]
I re-triggered the CircleCI tests manually, the integration RHEL tests seems to have a general issue today.

**Comment by mrunalp** [comment]
@nalind Is this ready?

**Comment by mrunalp** [comment]
nvm, I see that we need to switch to `github.com/containers/image`.

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@haircommander @mrunalp @saschagrunert @umohnani8 PTAL

File: go.mod [hunk 1]
```diff
@@ -63,6 +63,7 @@ require (
 )
 
 replace (
+	github.com/containers/libpod => github.com/nalind/libpod v0.0.0-20191028183138-999c4b94302e
```
Comment (saschagrunert) [inline go.mod]: Should this be removed?

Comment (haircommander) [reply go.mod]: ah good catch

**Comment by mrunalp** [comment]
The e2e-aws tests are running

**Pull request merged**